packages/graphback-datasync/src/providers/DatasyncMongoDBDataProvider.ts

```diff
@@ -56,7 +56,7 @@ export class DataSyncMongoDBDataProvider<Type = any> extends MongoDBDataProvider
 
     data._deleted = true;
```

Comment:
` this.fieldTransformMap[TransformType.UPDATE]` above? Why we doing this?

Reply:
This is to set the updatedAt timestamp

Reply:
Ah.. Looks fairly generic mapper for each field. We will need to see how this affects performance